tests/function_helpers_test.go

```diff
@@ -76,3 +77,19 @@ func scaleFunction(t *testing.T, name string, count int) {
 		t.Fatalf("scale got %d, wanted %d (or %d)", res.StatusCode, http.StatusAccepted, http.StatusOK)
 	}
 }
+
+func copyNamespacesTest(cases []FunctionMetaSchema) []FunctionMetaSchema {
```

Comment:
this is a nitpick, but i wonder if something like `FunctionTestCase` might be a better name for this type

Reply:
The function is used to replicate testcase for another namespace. possibly correct name should be `replicateTestCaseForNamespace` with function declaration as `func replicateTestCaseforNamespace(cases []FunctionMetaSchema, namespace string) []FunctionMetaSchema`

Reply:
I actually mean `FunctionMetaSchema` becoming something like `FunctionTestCase`, the function name seems fine.

Reply:
@LucasRoesler refactored the code to create single list for all the test cases. Please have a look at it.